website/client-old/js/controllers/menuCtrl.js

```diff
@@ -28,7 +29,7 @@ angular.module('habitrpg')
           } else if (groupNotificationTypes.indexOf('GROUP_TASK_APPROVED') !== -1) {
             return groupApproved;
           }
-          return noneValue;
+          return noneValue;s
```

Comment:
Yep :P Thanks. I noticed it locally. Will push a fix soon and any other feedback I received by tomorrow.